source/whereAny.js

```diff
@@ -0,0 +1,52 @@
+import _curry2 from './internal/_curry2';
+import _has from './internal/_has';
+import _isEmpty from './isEmpty';
+
+
+/**
+ * Takes a spec object and a test object; returns true if the test satisfies
+ * the spec. Each of the spec's own properties must be a predicate function.
+ * Each predicate is applied to the value of the corresponding property of the
+ * test object. `whereAny` returns true if at least one of the predicates return true, false
+ * otherwise.
+ *
+ * `whereAny` is well suited to declaratively expressing constraints for other
+ * functions such as [`filter`](#filter) and [`find`](#find).
+ *
+ * @func
+ * @memberOf R
+ * @since v0.27.0
```

Comment:
To be defined

Reply:
Yes, the build process should add this automatically, so let's take it out.

Reply:
- [ ] rely on automatic versioning